keyboards/boardsource/equals/48/config.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 Cole Smith (@boardsource)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 200U
```

Comment:
This feels ... very short, and I'm not sure how well this short of a timeout will work.

Reply:
I have tested it with leds and screen going and without and it works both times so seems go to me. I am happy to change it to any value you would like tho.